### PR TITLE
codecov raise threshold for commit status

### PR DESCRIPTION
**Pull request by laerus**

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+coverage:
+  status:
+    project:
+      default:
+        threshold: 1
```
Comment (astronouth7303) [inline codecov.yml]: Unit?

Comment (laerus) [inline codecov.yml]: yeah i had it at first then codecov docs made me change that
https://docs.codecov.io/docs/commit-status#section-threshold
ambiguity wins here

Comment (astronouth7303) [inline codecov.yml]: So it's in percentage points.

Comment (astronouth7303) [inline codecov.yml]: (YAML allows comments)

**Comment by scopatz** [comment]
Thanks @laerus! I really appreciate it!

**Pull request merged**